sormas-ui/src/main/java/de/symeda/sormas/ui/utils/components/sidecomponent/SideComponent.java

```diff
@@ -81,4 +80,16 @@ public Registration addSideComponentEditEventListener(SideComponentEditEventList
 			sideComponentEditEventListener,
 			SideComponentEditEventListener.ON_SIDE_COMPONENT_EDIT_METHOD);
 	}
+
+	private boolean userHasRight(UserRight... userRights) {
+		UserProvider currentUser = UserProvider.getCurrent();
+		if (currentUser != null) {
```

Comment:
You can basically replace the whole method with one single line of code:

`return Arrays.asList(userRights).stream().anyMatch(r -> currentUser.hasUserRight(r));`

I don't think the null check is needed because there always HAS to be a current user; if that's not the case, we have a problem anyway.